src/libs/tmdk/src/commands/nuke.ts

```diff
@@ -36,9 +39,27 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
     },
   });
 
+/**
+ * Helper function during workspace nuke for determining existance of workspace
+ * @param workspaceId TM workspace
+ * @returns promise boolean
+ */
+async function workspaceExists(workspaceId: string) {
+  try {
+    await aws().tm.getWorkspace({ workspaceId: workspaceId as string });
+    return true;
+  } catch (e) {
+    if (e instanceof ResourceNotFoundException) {
+      return false;
+    } else {
+      throw new Error(`Failed to get workspace. ${e}`);
+    }
+  }
+}
+
```

Comment:
This method can get moved to utils and used in other commands, can do in later pr, but please add a TODO

Reply:
I'll add the TODO, I moved it here bc we removed Utils.ts bc it had lots of unused code that had TS problems